CHANGELOG.md

```diff
@@ -0,0 +1,889 @@
+```git
+-    "fh-db": "2.0.0",
++    "fh-db": "3.0.0",
+```
+
+
+## [7.0.16] - Mon Jul 17 15:55:53 2017 +0100
+### Changed
+- updates dependencies:
+```git
+-    "fh-sync": "^1.0.4",
++    "fh-sync": "^1.0.5",
+```
+
+
+## [7.0.15] - Fri Jul 14 16:49:18 2017 +0100
+### Changed
+- Reverted to 2.0.0 due to breaking API changes in the minor version bump to 2.1.0
+```git
+-    "fh-db": "2.1.0",
++    "fh-db": "2.0.0", 
+```
+
+
+## [7.0.14] - Fri Jul 14 14:35:28 2017 +0100
+### Changed
+- updates dependencies:
+```git
+-    "fh-security": "0.2.0",
++    "fh-security": "0.2.1",
+```
+
+
+## [7.0.13] - Tue Jul 11 14:38:08 2017 +0100
+### Changed
+- This updates the version of the fh-sync module to include a fix for the ttl of queue messages. See https://github.com/feedhenry/fh-sync/pull/22 for more info.
+- updates dependencies:
+```git
+-    "fh-sync": "1.0.3",
++    "fh-sync": "^1.0.4",
+```
+
+### Removed
+- removed dependencies no longer needed:
+```git
+-    "backoff": "^2.5.0",
+-    "colors": "0.6.2",
+-    "cycle": "1.0.3",
+-    "eyes": "0.1.8",
+-    "fh-component-metrics": "2.7.0",
+-    "moment": "2.15.2",
+-    "mongodb": "2.1.18",
+-    "mongodb-lock": "0.4.0",
+-    "mongodb-queue": "2.2.0",
+-    "parse-duration": "0.1.1",
+-    "pkginfo": "0.2.3",
+-    "stack-trace": "0.0.9",
+```
+
+
+## [7.0.12] - Fri May 26 11:03:33 2017 +0100
+### Changed
+- Breaking API changes were introduced in the fh.db list call. Should have been a major version bump on fh.db
+```git
+-    "fh-db": "2.0.0",
++    "fh-db": "2.1.0", 
+```
+
+
+## [7.0.11] - Tue Jun 6 15:18:35 2017 +0100
+### Changed
+- removes sync api prefix
+- updates dependencies:
+```git
+-    "fh-sync": "~1.0.0",
++    "fh-sync": "~1.0.1",
+```
+
+
+## [7.0.10] - Tue May 23 11:31:11 2017 +0100
+### Added
+- Added MONGODB_VERSION env vars for MongoDB 2.4 & 3.2 to the travis build matrix 
+
+### Changed
+- Update to ensure an update object is passed in for tests
+- updates dependencies:
+```git
+-    "fh-sync": "^1.0.0-RC1",
++    "fh-sync": "~1.0.0",
+```
+
+### Removed
+- Removed sync source code & refactored fh-mbaas-api to now use decoupled fh-sync library instead
+
+
+## [7.0.9] - Tue May 16 13:05:23 2017 +0100
+### Changed
+- Fixed background typo
+
+
+## [7.0.8] - Tue May 16 09:41:47 2017 +0100
+### Changed
+- Uses stringify for error messages
+- Returns sync error to conect() requests by sync clients if sync not ready to start; 'crashed' notification sent to client and client will retry later.
+- Corrects null error (FH-3591)
+- Change the default sync worker interval & backoff settings to take full advantage of CPU when the sync queue is busy and to allow the CPU to be used for other things when the sync queue is not busy.
+
+### Removed
+- Removes 'stop' test
+
+
+## [7.0.7] - Fri May 12 15:42:56 2017 +0100
+### Changed
+- Uses mongodb ttl to remove queue messages
+- Makes sure only 1 cleaner job will remove the data when there are multiple workers
```

Comment:
Docs team use `one` instead of `1`

Reply:
@austincunningham good point, although not sure we can count on enforcing these conventions for upstream documentation all of the time. 

I think it could be worth highlighting good docs conventions in our README's though, especially as we are trying to enforce a standard convention for commit messages.